iOS/Flows/Conversation/Conversation List/ConversationListCoordinator.swift

```diff
@@ -111,3 +113,47 @@ class ConversationListCoordinator: InputHandlerCoordinator<Void> {
         ConversationsManager.shared.activeConversation = controller.conversation
     }
 }
+
+// MARK: - Permissions Flow
+
+extension ConversationListCoordinator {
+
+    @MainActor
+    func checkForPermissions() async {
+        if INFocusStatusCenter.default.authorizationStatus != .authorized {
+            self.presentPermissions()
+        } else if await UserNotificationManager.shared.getNotificationSettings().authorizationStatus != .authorized {
+            self.presentPermissions()
+        }
+    }
+
+    @MainActor
+    private func presentPermissions() {
+        let coordinator = PermissionsCoordinator(router: self.router, deepLink: self.deepLink)
+        self.present(coordinator, finishedHandler: nil, cancelHandler: nil)
+    }
+}
+
+// MARK: - Image View Flow
+
+extension ConversationListCoordinator: LightboxControllerDismissalDelegate {
+
+    func presentImageFlow(for imageURL: URL) {
+        let images = [LightboxImage(imageURL: imageURL)]
+
+        // Create an instance of LightboxController.
+        let controller = LightboxController(images: images)
+
+        // Set delegates.
+        controller.dismissalDelegate = self
+
+        // Use dynamic background.
+        controller.dynamicBackground = true
+
+        self.listVC.present(controller, animated: true)
+    }
```

Comment:
This isn't connected yet, but it's the basic way you present an image with the lightbox pod.

Reply:
I think you'll want to add all of this to the new InputHandlerCoordinator if its going to be shared by threads.

Reply:
Ok will do